src/ui/components/SearchInput/style.scss

```diff
@@ -46,6 +43,15 @@ $transition-duration: 250ms;
     opacity: 1;
     // Use !important to overwrite style properties added in JS.
     transform: translateX(0) !important;
+  }
+}
+
+.SearchInput--animating {
+  .SearchInput-animation-icon {
+    transition: opacity 1ms $transition-duration, transform $transition-duration;
```

Comment:
The first one is the duration, the second is the delay. So the icon will get hidden after `$transition-duration` which is when the label re-appears. I might be able to just set this on `SearchInput--animating` now since we have that class. I'll fix it or add a comment.